TAO/tao/default_resource.cpp

```diff
@@ -791,12 +791,14 @@ TAO_Default_Resource_Factory::get_reactor ()
   else
     this->dynamically_allocated_reactor_ = true;
 
+  ACE_Reactor::instance (reactor);
```

Comment:
Doesn't look a good thing to just replace the singleton reactor with the one created by TAO, something to revert, also reclaim

Reply:
Maybe TAO_DEFAULT_REACTOR_TYPE is a helpful define to change the default reactor type for TAO?

Reply:
TAO allows, or seems to allow, a different reactor to be configured for the ORB, separate from ACE's singleton reactor or any other reactor.

Reply:
Yes, but the code here still resets it, so something to remove

Reply:
We're working on it -- see https://github.com/DOCGroup/ACE_TAO/pull/2340#discussion_r1983330540